### PR TITLE
DAOS-8648 tests: Re-enable skipped tests

### PR DESCRIPTION
**Pull request by phender**
Removing skipForTicket decorator for tests being skipped by resolved tickets DAOS-7195, DAOS-7955, and DAOS-7493.

Quick-functional: true
Test-tags: nvme_pool_extend test_osa_offline_extend_oclass test_osa_offline_extend_during_aggregation test_osa_offline_parallel_test_with_aggregation online_extend test_osa_online_reintegration_server_stop

Required-githooks: true

Signed-off-by: Phil Henderson <phillip.henderson@intel.com>

### CODE REVIEW

**Review by rpadma2 — APPROVED** [review]

**Review by shimizukko — DISMISSED** [review]

**Review by daltonbohning — CHANGES_REQUESTED** [review]
-1 just for the introduced pylint warning

File: src/tests/ftest/osa/offline_parallel_test.py [hunk 1]
```diff
@@ -8,13 +7,13 @@
...
 import copy
+import queue
 from osa_utils import OSAUtils
 from daos_utils import DaosCommand
 from dmg_utils import check_system_query_status
 from exception_utils import CommandFailure
 from test_utils_pool import add_pool
-from apricot import skipForTicket
-import queue
+
```
[the remark this answers is not in the document]
Comment (phender) [reply src/tests/ftest/osa/offline_parallel_test.py]: Fixed

**Review by daltonbohning — DISMISSED** [review]

**Comment by phender** [comment]
In https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10275/3 the OSAOfflineExtend, OSAOfflineParallelTest, and OSAOnlineReintegration tests passed.  The NvmePoolExtend and OSAOnlineExtend tests failed.

**Review by rpadma2 — DISMISSED** [review]

**Review by shimizukko — APPROVED** [review]

File: src/tests/ftest/nvme/pool_extend.py [hunk 2]
```diff
@@ -26,107 +24,121 @@ class NvmePoolExtend(OSAUtils):
...
 
-        for val in range(0, num_pool):
-            # Create a pool
-            self.pool.append(self.get_pool())
-            self.pool[-1].set_property("reclaim", "disabled")
+        # Create the pools
+        pools = []
+        for _ in range(0, num_pool):
+            pools.append(self.get_pool(properties="reclaim:disabled"))
+            # pools[-1].set_property("reclaim", "disabled")
```
Comment (daltonbohning) [inline src/tests/ftest/nvme/pool_extend.py:58]: Remove the comment?

File: src/tests/ftest/util/test_utils_pool.py [hunk 3]
```diff
@@ -1072,3 +1074,66 @@ def measure_rebuild_time(self, operation, interval=1):
         self.wait_for_rebuild(to_start=False, interval=interval)
         duration = float(time()) - start
         self.log.info("%s duration: %.1f sec", operation, duration)
+
+    @fail_on(CommandFailure)
+    def extend(self, ranks):
+        """Extend the pool to additional ranks.
+
+        Args:
+            ranks (list): a list daos server ranks (int) to exclude
```
Comment (daltonbohning) [inline src/tests/ftest/util/test_utils_pool.py:1083]:
```suggestion
            ranks (list): a list of daos server ranks (int) to exclude
```

File: src/tests/ftest/util/test_utils_pool.py [hunk 4]
```diff
@@ -1072,3 +1074,66 @@ def measure_rebuild_time(self, operation, interval=1):
...
+
+        """
+        return self.dmg.pool_extend(self.identifier, ranks)
```
Comment (daltonbohning) [inline src/tests/ftest/util/test_utils_pool.py:1086]: `Returns` docstring?

**Review by daltonbohning — APPROVED** [review]
LGTM - minor comments

**Comment by phender** [comment]
All tests passed in https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10275/6/testReport/

**Pull request merged**